src/main/java/uk/org/openbanking/datamodel/fund/OBFundsConfirmationConsentResponse1Data.java

```diff
@@ -35,239 +34,250 @@
 /**
  * OBFundsConfirmationConsentResponse1Data
  */
-@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-05-20T10:02:34.555654+01:00[Europe/London]")
-
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-11-18T09:57:44.731259Z[Europe/London]")
 public class OBFundsConfirmationConsentResponse1Data {
-  @JsonProperty("ConsentId")
-  private String consentId;
+    @JsonProperty("ConsentId")
+    private String consentId;
+
+    @JsonProperty("CreationDateTime")
+    @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
+    private DateTime creationDateTime;
 
-  @JsonProperty("CreationDateTime")
-  @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
-  private DateTime creationDateTime;
+    /**
+     * Specifies the status of consent resource in code form.
+     */
+    public enum StatusEnum {
+        AUTHORISED("Authorised"),
 
-  /**
-   * Specifies the status of consent resource in code form.
-   */
-  public enum StatusEnum {
-    AUTHORISED("Authorised"),
+        AWAITINGAUTHORISATION("AwaitingAuthorisation"),
 
-    AWAITINGAUTHORISATION("AwaitingAuthorisation"),
+        REJECTED("Rejected"),
 
-    REJECTED("Rejected"),
+        REVOKED("Revoked");
 
-    REVOKED("Revoked");
+        private String value;
 
-    private String value;
+        StatusEnum(String value) {
+            this.value = value;
+        }
+
+        @JsonValue
+        public String getValue() {
+            return value;
+        }
 
-    StatusEnum(String value) {
-      this.value = value;
+        @Override
+        public String toString() {
+            return String.valueOf(value);
+        }
+
+        @JsonCreator
+        public static StatusEnum fromValue(String value) {
+            for (StatusEnum b : StatusEnum.values()) {
+                if (b.value.equals(value)) {
+                    return b;
+                }
+            }
+            throw new IllegalArgumentException("Unexpected value '" + value + "'");
+        }
     }
 
-    @JsonValue
-    public String getValue() {
-      return value;
+    @JsonProperty("Status")
+    private StatusEnum status;
+
+    @JsonProperty("StatusUpdateDateTime")
+    @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
+    private DateTime statusUpdateDateTime;
+
+    @JsonProperty("ExpirationDateTime")
+    @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
```

Comment:
Wouldn't better use a import?

Reply:
That's how it was generated, so I didn't want to mess with it or next time we generate it there would be a difference to investigate...